src/assets/theme/chart-theme.json

```diff
@@ -20,6 +20,9 @@
             "color": "rgba(255,255,255,0.5)"
         }
     },
+    "grid": {
+        "containLabel": true
+    },
```

Comment:
Prevents the label from getting cropped like this 
![image](https://github.com/global-water-watch/global-water-watch-website/assets/15196342/da65874e-40c3-4a3e-84c9-00fbbf0ff616)